app/domain/authentication/authn_jwt/identity_providers/identity_from_url_provider.rb

```diff
@@ -2,7 +2,7 @@ module Authentication
   module AuthnJwt
     module IdentityProviders
       # Provides jwt identity from information in the URL
-      class IdentityFromUrlProvider < IdentityProviderInterface
+      class IdentityFromUrlProvider
```

Comment:
this can definitely be a command class called `FetchJwtIdentity` as this is the only function

Reply:
It cant because FetchIdentityFromDecoded token can't be command class and i want same singature for these objects

Reply:
status api needs this function in identity_from_decoded_token_provider
```
def validate_identity_configured_properly
          return unless identity_available?

          validate_token_id_field_has_value
          validate_identity_path_configured_properly
        end
```

Reply:
why do we need the classes to have the same signature?

Reply:
Because they from the same factory. Nevermind i splited the IdentityFromDecoded token to 2 classes and made these two command classes